platform-sdk/docs/proposals/consensus-layer/Consensus-Layer.md

```diff
@@ -0,0 +1,669 @@
+# Consensus Layer of the Consensus Node
+
+---
+
+## Summary
+
+Update the architecture for the consensus node to reduce complexity, improve performance, and improve stability.
+
+| Metadata           | Entities                                                 | 
+|--------------------|----------------------------------------------------------|
+| Designers          | Richard Bair, Jasper Potts, Oleg Mazurov, Austin Littley |
+| Functional Impacts | Consensus Node                                           |
+
+---
+
```

Comment:
Some assumptions up top would be useful. For example, this proposal assumes that the definition of ancient is based on birth rounds, not generations.

Reply:
I created a section for assumptions, and added this one.

No other assumptions that need to be called out come to mind, but if any one has suggestions for additional points, I can add those as well.